docs/release-instruction.md

```diff
@@ -43,6 +43,11 @@ Write new revision number into the root `VERSION` file:
 $ echo ${NEOFS_TAG_PREFIX}${NEOFS_REVISION} > VERSION
 ```
 
+Update version in Debian package changelog file
+```
+$ cat debian/changelog
+```
+
```

Comment:
How cat helps to update version? Why not to use dch?

Reply:
That's not a real package changelog, but a placeholder to let build the package with common tools. So with new release we just make sure the default version set there matches the release version.